### ISSUE TITLE
Error while using the generator on mac

### ISSUE THREAD

**Issue by ashutoshraina** [ligershark/generator-aspnet#23]
```
? What type of application do you want to create? MVC Application
? What's the name of your ASP.NET application? Awesome

     info ... Fetching http://github.com/ligershark/aspnet_vnext_samples/archive/master.tar.gz ...
     info This might take a few moments
..✔ Done in /Users/ashutoshraina/.cache/yeoman/ligershark/aspnet_vnext_samples/master

   create NuGet.config
   create global.json
   create Awesome/Controllers/HomeController.cs
   create Awesome/Models/User.cs
   create Awesome/Startup.cs
   create Awesome/Views/Home/Index.cshtml
   create Awesome/Views/Shared/_Layout.cshtml
   create Awesome/project.json
   create Awesome/wwwroot/.gitignore
/Users/ashutoshraina/.cache/yeoman

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: spawn ENOENT
    at errnoException (child_process.js:1001:11)
    at Process.ChildProcess._handle.onexit (child_process.js:792:34)
```

Let me know if I can provide some other information that might be useful.

**Comment by sayedihashimi** [comment]
After the project is created `kpm restore` is kicked off which is a part of the [ASP.NET vNext command line tools](https://github.com/aspnet/home). On Mac if you start a new terminal to make the kpm commands available you have to invoke `source kvm.sh`.

The error above is the error you'll get if `kvm` isn't available. So it's likely one of the following.
1. [ASP.NET vNext command line tools](https://github.com/aspnet/home) are not installed
2. You need to execute `source kvm.sh`

FYI at #21 we are discussing to turn this off. We are going to let uses invoke `kpm restore` whenever they are ready instead of right after. So after fixing that we will avoid this issue all together.

I'll close this item now, but if the notes above don't address your issue please re-activate.

**Comment by ashutoshraina** [comment]
That seems to be the case, thanks for the clarification.

**Comment by sayedihashimi** [comment]
OK I'm removed the call to `kpm restore` and published to npmjs as version `0.0.11`. Let me know if you have any issues with the updates.
